content/ai/concepts/neural-networks/terms/artificial-neural-networks/artificial-neural-networks.md

```diff
@@ -0,0 +1,42 @@
+---
+Title: 'Artificial Neural Networks'
+Description: 'A neural network is a method in artificial intelligence that teaches computers to process data in a way that is inspired by the human brain.'
+Subjects:
+  - 'Machine Learning'
```

Comment:
The subjects you've added are more appropriate as tags. This is the suggested subjects for this entry:
```
  - 'Machine Learning'
  - 'Computer Science'
```

Reply:
@SSwiniarski copy that, should I deleted the rest of the subjects & leave only those 2? 
And also is it ok if I delete the other 2 branches?